samples/EnableValidationWithCallback/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ set(SOURCES
 source_group(headers FILES ${HEADERS})
 source_group(sources FILES ${SOURCES})
 
+add_definitions(-DVULKAN_HPP_DISPATCH_LOADER_DYNAMIC=0)
```

Comment:
>add_definitions(-DVULKAN_HPP_DISPATCH_LOADER_DYNAMIC=0)
Those definitions are not required.

Reply:
They are needed, as the samples/CMakeLists.txt adds that definition with a 1. There are a few samples that don't need the dynamic DispatchLoader.